packages/web/pages/player/[username].tsx

```diff
@@ -100,6 +101,15 @@ const PlayerPage: React.FC<Props> = ({ player }) => {
 
   return (
     <PageContainer>
+      <HeadComponent
+        title={`Metagame profile for ${player.username}`}
+        description={`${player.box_profile?.description} || ''`.replace(
+          '\n',
+          ' ',
+        )}
+        url={`https://my.metagame.wtf/player/${player.username}`}
+        img={player.box_profile?.imageUrl || ''}
```

Comment:
Here also you can use the `getPlayerImage` helper

Reply:
resolved by d9a4e2a